authui/check-bundle-consistency.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# This script is used to test whether the built bundles are having the same hash over several builds
+
+N=5
+MANIFEST_LOCATION=../resources/authgear/generated/manifest.json
+
+if [ $N -lt 2 ]; then
+  echo "[ERROR] The build number must be greater than 1"
+  exit 1
+fi
+
+build_bundles()
+{
+  for i in $(seq 1 $N)
+  do
+    echo "[INFO] Build no.$i"
+
+    # Clear vite cache before building bundles
+    rm -rf node_modules/.vite > /dev/null 2>&1
+    npm run build > /dev/null 2>&1
+
+    jq --sort-keys . $MANIFEST_LOCATION > $i.json
```

Comment:
You do not write things in our project. You write to a temporary directory.

Since you write to a temporary directory, you do not need to clean it up. We expect it will be cleaned up by somebody else.

To create a temporary directory, you use `mktemp -d`. Please `man mktemp` to see its usage.

Reply:
ok!